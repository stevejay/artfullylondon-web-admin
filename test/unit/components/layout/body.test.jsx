import React from 'react'
import { shallow } from 'enzyme'

import Body from '_admin/components/layout/body'

it('should render correctly', () => {
  const wrapper = shallow(
    <Body>
      <div id='child' />
    </Body>
  )

  expect(wrapper).toMatchSnapshot()
})
