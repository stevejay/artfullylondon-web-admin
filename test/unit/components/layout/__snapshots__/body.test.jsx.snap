// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="src-components-layout-body__container"
>
  <div
    id="child"
  />
</div>
`;
